File: Python/Lab1/Variant18.py
from math import *
a = float(input("Введите значение a: "))
b = float(input("Введите значение b: "))
x = float(input("Введите значение x: "))
d = a*exp(-sqrt(a))*cos(b*x/a)
print("\nЗначение d =", d)

File: Pascal/Lab1/Variant18.pas
Program lab1;
Uses Crt;
Var a,b,x,Formula: real;
Begin
  Write('Введите первое число: '); Readln(a);
  Write('Введите второе число: '); Readln(b);
  Write('Введите третье число: '); Readln(x);
  ClrScr;
  Formula:=a*exp(-sqrt(a))*cos((b*x)/a);
  Writeln('Вариант 18.  ',Formula);
End.